src/deploy/functions/runtimes/python/index.ts

```diff
@@ -0,0 +1,184 @@
+import fetch from "node-fetch";
+import * as path from "path";
+import * as fs from "fs";
+import * as portfinder from "portfinder";
+import * as spawn from "cross-spawn";
+import * as cp from "child_process";
+
+import * as runtimes from "..";
+import * as backend from "../../backend";
+import { logger } from "../../../../logger";
+import * as discovery from "../discovery";
+import { FirebaseError } from "../../../../error";
+import { Build } from "../../build";
+
+export const LATEST_VERSION: runtimes.Runtime = "python310";
```

Comment:
FYI latest is now python311

Reply:
Is it supported in GCF? Per https://cloud.google.com/functions/docs/concepts/python-runtime, I dont see python311.

Reply:
Hmm, maybe I jumped the gun I was looking a cs